server/package.json

```diff
@@ -10,19 +10,23 @@
   "scripts": {
     "test": "node --test test/**/*.test.js",
     "start": "fastify start -l info app.js",
-    "dev": "fastify start -w -l info -P app.js",
+    "dev": "npx prisma migrate dev && fastify start -w -l info -P app.js",
+    "migrate": "docker-compose exec server sh -c 'cd server && npx prisma migrate $1' sh",
+    "db": "docker-compose exec server sh -c 'cd server && npx prisma db $1' sh",
```

Comment:
Using `$1` will only pass the first argument to the script... I'd recommend changing this to `$@` which will pass _all_ arguments...

That way, you can execute commands that have additional params, like:

`npm run -w server migrate -- dev --name=...`

Reply:
good call